src/main/java/org/jetbrains/research/groups/ml_methods/Logging.java

```diff
@@ -0,0 +1,28 @@
+package org.jetbrains.research.groups.ml_methods;
+
+import org.apache.log4j.*;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Logging {
+    @NotNull
+    private static final Path LOG_FILE_PATH = Paths.get("./log");
+
+    @NotNull
+    public static Logger getLogger(@NotNull Class<?> aClass) {
+        final Logger logger = Logger.getLogger(aClass);
+        logger.setLevel(Level.DEBUG);
+        try {
+            logger.addAppender(new FileAppender(new PatternLayout("%p [%c.%M] - %m%n"),
```

Comment:
This log appends to file which means that log file contains logs for all runs that were performed. It is very inconvenient to read such log files. I suggest to configure it to rewrite file, not append to it.

Reply:
Implemented.